modules/rubiconAnalyticsAdapter.js

```diff
@@ -238,6 +256,21 @@ function parseBidResponse(bid) {
 
 let samplingFactor = 1;
 let accountId;
+// List of known rubicon aliases
+// This gets updated on auction init to account for any custom aliases present
+let rubiconAliases = ['rubicon'];
+
+/*
+  Checks the alias registry for any entries of the rubicon bid adapter.
+  adds to the rubiconAliases list if found
+*/
+function setRubiconAliases(aliasRegistry) {
+  Object.keys(aliasRegistry).forEach(function (alias) {
+    if (aliasRegistry[alias] && aliasRegistry[alias] === 'rubicon') {
```

Comment:
you shouldn't need the first check `aliasRegistry[alias]`

Reply:
Oh yeah duh, being too careful.

Will remove.